From 26b4bc6b5d64096f565a3bf1aea89d8657956b5e Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 10 Jul 2019 18:03:34 -0700
Subject: [PATCH] TRITON-580 urclient could include some more details about
 who's running the command Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 71252cb..13a1a8f 100644
--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
         "json": "9.0.3",
         "jsprim": "0.5.1",
         "tabula": "1.4.1",
-        "urclient": "git+https://github.com/joyent/node-urclient.git#2b9f739",
+        "urclient": "1.2.1",
         "vasync": "1.5.0"
     },
     "devDependencies": {
-- 
2.21.0

